### PR TITLE
Add inverse transform in transformer

### PR DESCRIPTION
**Pull request by vinx13**

### CODE REVIEW

File: src/interfaces/swig/Transformer.i [hunk 1]
```diff
@@ -5,7 +5,8 @@
  */
 
 /* These functions return new Objects */
-%newobject shogun::CTransformer::transform(CFeatures*);
+%newobject shogun::CTransformer::transform(CFeatures*, bool inplace=true);
+%newobject shogun::CTransformer::inverse_transform(CFeatures*, bool inplace=true);
```
Comment (karlnapf) [inline src/interfaces/swig/Transformer.i]: can we rename to `inverse` ... without `transform` ?

**Comment by vigsterkr** [comment]
The idea here was about sklearnish approach

**Comment by karlnapf** [comment]
I see they have that ... well, then it is probably useful. ... ;)

**Comment by vigsterkr** [comment]
Min max scaled target (regression) for which u learn a model but then u wanna actually map back the prediction into the “original scale”

**Comment by karlnapf** [comment]
Ah yes of course!
@vinx13 hey what about a cookbook for that use case? :)

**Pull request merged**